docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/are-zed-endpoints-split-to-3-backendapi-backendgateway-backoffice.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Are ZED endpoints split to 3 (BackendApi, BackendGateway, Backoffice)?
+description: This document allows you to assess if ZED endpoints are split to 3 (BackendApi, BackendGateway, Backoffice).
+template: howto-guide-template
+---
+
+# Are ZED endpoints split to 3 (BackendApi, BackendGateway, Backoffice)?
+
+{% info_block infoBox %}
+
+Resources: Backend
+
+{% endinfo_block %}
+
+## Description
+
+1. Navigate to `public` folder in the root of the project, if you don’t see **BackendApi**, **BackendGateway**, and **Backoffice**
+    folders with `index.php` entry points inside then it means endpoints are not split.
+2. Ensure this [integration guide](/docs/scos/dev/technical-enhancement-integration-guides/integrating-separate-endpoint-bootstraps.html#update-modules-using-composer) is implemented.
```

Comment:
What if it's not integrated?

Reply:
Doesn't it need to be in the formula then?

Reply:
The formula says "Approximately 1d if required packages are already installed". It is about the integration guide. In case some packages require an update, it must be done in the "Is project based on or updated to the 202108.0 demoshop release?" step, it has its own formula.

Reply:
Still quite vague. 

1. What packages are we referring to in the formula? 
2. If the project fulfills both requirements, what are we going to spend 1d for?

Reply:
1. List of packages we have in the `integration guide` and also in the "Is project based on or updated to the 202108.0 demoshop release?" document.
2. If the project fulfills both requirements, then there is nothing to do. All the formulas are only for the migration phase and not for assessment so if the project fits all requirements it means we don't need to spend any time.

Reply:
The formula says "1d if the packages *are installed*". But you say that if the packages from the integration guide are installed, there is nothing else to do. 

Does it make sense to change the formula to remove the condition `if the packages are installed`. It is quite confusing as it is right now

Reply:
I see a misunderstanding here. 🙂  You asked "If the project fulfills both requirements, what are we going to spend 1d for?" and I answered that in this case there is nothing to do, but in the case when all required packages are installed but the integration guide is not implemented, we need 1 PD to implement the integration guide.